python/ray/workers/default_worker.py

```diff
@@ -80,6 +80,11 @@
     default=False,
     action="store_true",
     help="True if cloudpickle should be used for serialization.")
+parser.add_argument(
```

Comment:
We don't need to expose this because there will be no external components that ping this process. It will be pinged only through localhost.

Reply:
This is the core of this PR. It's used by Raylet to pass the agent port to newly started worker.

Reply:
oh my bad. I thought it was `ray start`. Totally makes sense.